Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG CUDA_VER=11.8.0
+ARG PYTHON_VER=3.10
+ARG LINUX_VER=ubuntu20.04
+
+FROM rapidsai/mambaforge-cuda:cuda${CUDA_VER}-base-${LINUX_VER}-py${PYTHON_VER} as base
```

Comment:
This is a minimal repo?  I'm guessing it is, but the rapidsai prefix scares me a little :)

Reply:
Yeah, that's correct. 

This images is just the `nvidia/cuda` image + `mambaforge` installed. FWIW, with the extra packages we're adding here, the `coiled/gpu` image is about 2x smaller than the base rapids image.

Reply:
+1